synapse/storage/databases/main/room.py

```diff
@@ -1411,6 +1411,54 @@ async def add_event_report(
             desc="add_event_report",
         )
 
+    async def get_event_report(self, report_id: int) -> Dict[str, Any]:
```

Comment:
```suggestion
    async def get_event_report(self, report_id: int) -> Optional[Dict[str, Any]]:
```

Since we potentially expect `None` to be returned from `await self.store.get_event_report(report_id)` in `event_reports.py`.